src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -41,14 +41,14 @@ internal sealed class MainModuleResolution : ModuleResolutionBase, IModuleManage
         private readonly IUIService _ui;
         private IRunningDocumentTable _rdt;
 
-        private ImmutableArray<string> _userPaths = ImmutableArray<string>.Empty;
+        private ImmutableArray<string> _userConfiguredPaths;
 
-        public MainModuleResolution(string root, IServiceContainer services)
+        public MainModuleResolution(string root, IServiceContainer services, ImmutableArray<string> userConfiguredPaths = default)
```

Comment:
I'd rather not have this parameter optional.

Reply:
Oops, that was a leftover. Removed.